src/validate.rs

```diff
@@ -108,7 +108,7 @@ pub fn validate_duration(block: &Block, data: &Everything, sc: &mut ScopeContext
 pub fn validate_optional_duration_int(vd: &mut Validator) {
     let mut count = 0;
 
-    for field in &["days", "weeks", "months", "years"] {
+    for field in &["days", "weeks", "months", "years", "duration"] {
```

Comment:
It is used in all of the add_X_modifier effects for imperator. In practice it functions the exact same as "days" but for some odd reason the imperator devs implemented it as "duration" instead.

Reply:
Ok :)

Hmm I'd rather not accept `duration` everywhere because it's probably not valid for the other games or even other duration blocks in Imperator. (Might be worth verifying though)

I think it's ok to make a separate function for this. We already have four, one more won't hurt :)